### PR TITLE
[doc][yba] Update Manage Users page

### PR DESCRIPTION
**Pull request by ddhodge**
Update Manage Users page to include both classic and fine grain RBAC

@netlify /stable/yugabyte-platform/administer-yugabyte-platform/anywhere-rbac/

### CODE REVIEW

**Review by streddy-yb — APPROVED** [review]
@subramanian-neelakantan - can you also review/approve?

File: docs/content/stable/yugabyte-platform/administer-yugabyte-platform/anywhere-rbac.md [hunk 1]
```diff
@@ -29,17 +30,88 @@ curl --request PUT \
...
+- **Admin** has privileges that are similar to the Super Admin, except that Admin cannot manage global scope artifacts and actions, such as runtime configuration settings and LDAP authentication.
+
+- **Backup Admin** has access to backup-related tasks, such as the following:
+
+  - Manage database backups and restore operations.
+  - Create new backups.
+  - Delete any existing backup or backup schedule.
+  - Edit existing backups.
+
+  Backup Admin has View permissions for all other resources.
```
Comment (aishwarya24) [inline docs/content/stable/yugabyte-platform/administer-yugabyte-platform/anywhere-rbac.md:67]:
```suggestion
  Backup Admin has view permissions for all other resources.
```

File: docs/content/stable/yugabyte-platform/administer-yugabyte-platform/anywhere-rbac.md [hunk 2]
```diff
@@ -29,17 +30,88 @@ curl --request PUT \
...
+
+1. Select a [built-in role](#built-in-roles) for the user.
+
+1. Click **Submit**.
+
+To modify a user role or delete the user, navigate to **Admin > User Management > Users**. Click **Actions** that corresponds to the specific user, and then select either **Edit User Role** or **Delete User**.
+
+  </div>
+
+  <div id="finegrain" class="tab-pane fade" role="tabpanel" aria-labelledby="finegrain-tab">
```
Comment (aishwarya24) [inline docs/content/stable/yugabyte-platform/administer-yugabyte-platform/anywhere-rbac.md:115]: felt that you can mention about the version part again that fine-grained RBAC is available from 2.20.1 just in brackets where you the tabs maybe.

**Review by aishwarya24 — APPROVED** [review]
LGTM

**Review by subramanian-neelakantan — APPROVED** [review]
Looks good to me. Thanks!

**Pull request merged**